### PR TITLE
migrate styled-system

### PR DESCRIPTION
**Pull request by Kikobeats**
**to review**

- [x] home
- [x] styleguide
- [x] recipes
- [x] recipes/:id
- [x] privacy
- [x] tos
- [x] blog
- [x] blog/:id
- [x] docs
- [x] dpa
- [x] changelog
- [x] community
- [x] design
- [x] enterprise
- [x] formats
- [x] insights
- [x] logo
- [x] meta
- [x] newsletter
- [x] oss
- [x] pdf
- [x] screenshot
- [x] sdk
- [x] status
- [x] user-agents
- [x] payment
- [x] payment/update

### CODE REVIEW

**Review by The-Code-Monkey — COMMENTED** [review]
I would still recommend using compose as it speeds up runtime

**Comment by The-Code-Monkey** [comment]
Just checking to see if all is going well, any issues?

**Comment by Kikobeats** [comment]
@The-Code-Monkey Thanks for asking!

There is still a lot of work to be done, but I will do it little by little.

I can feedback you about https://styled-system-docs.vercel.app. In general terms amazing job, all is working as expected. Some minor issues:

- There is `styled-system` references instead of `@techstack/styled-system` ([example](https://styled-system-docs.vercel.app/reference-table#typography))
- I believe [inline variants](https://github.com/styled-system/styled-system/blob/master/docs/variants.md) are not implemented. I'm not totally sure but maybe that way of variant will do the system a bit lightweight vs. keeping all the components variants in memory all the time?

**Comment by The-Code-Monkey** [comment]
In terms of docs I'll sort that at some point, for variants I like to define them in the theme file rather then online not much you can do if they are inline

File: src/components/elements/Container.js [hunk 1]
```diff
@@ -1,14 +1,21 @@
...
+    pt: [5, 5, 6, 6],
+    px: [3, 3, 0, 0]
+  })}
+`
 
 Container.defaultProps = {
-  flexDirection: 'column',
-  ml: 'auto',
-  mr: 'auto',
-  pt: [5, 5, 6, 6],
```
Comment (breadadams) [inline src/components/elements/Container.js:11]: **reminder:** This was used in quite a few places (e.g. `Container.defaultProps.pt`), need to replace with relevant value.

**Comment by Kikobeats** [comment]
@breadadams @The-Code-Monkey finally shipped 🎉

**Pull request merged**